langs/bash/Dockerfile

```diff
@@ -2,6 +2,8 @@ FROM alpine:3.20 AS builder
 
 RUN apk add --no-cache build-base curl
 
+WORKDIR /app
```

Comment:
This is what you mean, isn't it?

Reply:
Yep, not wild on the name though, maybe `/lang` would be clearer. Or `/build`, I dunno.

Reply:
> Yep, not wild on the name though, maybe `/lang` would be clearer. Or `/build`, I dunno.

Nitballs 10/10.